.cscs/ci.yml

```diff
@@ -0,0 +1,121 @@
+include:
```

Comment:
Do you plan to enforce the same requirements to the new repository as we've had on the old one? If yes, then please add copyright and license information.

Reply:
You're fast! Yes, definitely :) This config doesn't even do anything HPX-related yet, and will change still significantly. Once it's working I'll add the same to HPX as well. But I suspect it's going to take a while given that the underlying system is still a work in progress.

However, in the meantime, could you check if this loads for you: https://gitlab.com/cscs-ci/ci-testing/webhook-ci/hpx-local-4750890491460239/-/pipelines/432028314? If yes, the logs will be public once this is working.

Reply:
I just wanted to add that inspect will still check for these things like on STEllAR-GROUP/hpx.